src/main/java/Duke.java

```diff
@@ -1,234 +1,53 @@
-            case "event":
-                if (input.length == 1) {
-                    throw new DukeException("☹ OOPS!!! The description of a " + input[0] + " cannot be empty.\n");
-                }
-                int i = 1;
-                for (; i < input.length; i++) {
-                    if (input[i].charAt(0) == '/') break;
-                    obj.append(input[i]);
-                    obj.append(" ");
-                }
-                if (i == input.length) {
-                    throw new DukeException("☹ OOPS!!! There is no proper date for " + input[0] + ".\n");
-                }
-                obj.setLength(obj.length()-1);
-                if (input[0].equals("deadline")) {
-                    todo.add(new DeadLine(obj.toString(), input[input.length-2], input[input.length-1]));
-                } else {
-                    todo.add(new Events(obj.toString(), input[input.length-3] ,input[input.length-2], input[input.length-1]));
-                }
-                System.out.println(Duke.speak(Dialogue.ADDED));
-                System.out.println(todo.get(todo.size()-1));
-                System.out.println(Duke.speak(Dialogue.NUMLEFT, todo.size()));
-                appendFile(filePath, todo.get(todo.size()-1));
-                break;
-            case "findDate":
-                if (input.length != 2) {
-                    throw new DukeException("Fill in proper integer to find date.\n");
-                }
-                for (Task task : todo) {
-                    if (task.sameTime(input[1])) {
-                        System.out.println(task);
-                    }
-                }
-                break;
-            default:
-                throw new DukeException("☹ OOPS!!! I'm sorry, but I don't know what that means :-(\n");
-        }
-    }
-
-    public static String speak(Dialogue option, Integer num) {
-        String reply;
-        switch (option) {
-            case NUMLEFT: reply =  "Now you have " + num.toString() + " task in the list\n";
-                break;
-            default: reply = "";
-        }
-        return reply;
-    }
-
-    public static String speak(Dialogue option) {
-        String reply;
-        switch (option) {
-            case GREETING: reply =  "Hello! I'm Duke.\nWhat can I do for you?";
-                break;
-            case MARK: reply = "Nice! I've marked this task as done:\n";
-                break;
-            case UNMARKED: reply = "OK, I've marked this task as not done yet:\n";
-                break;
-            case DELETE: reply = "Noted. I've removed this task:\n";
-                break;
-            case ADDED: reply = "Got it. I've added this task:\n";
-                break;
-            case LIST: reply = "Here are the tasks in your list:\n";
-                break;
-            case FAREWELL: reply = "Bye. Hope to see you again soon!\n";
-                break;
-            default: reply = "Are you finally done?";
-                break;
```

Comment:
What a clean and succinct way to check for the different conditions! I am impressed 😄 !